airflow/migrations/versions/83f031fd9f1c_change_ts_columns_to_datetime_on_mssql.py

```diff
@@ -0,0 +1,200 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""change ts/datetime columns to datetime/datetime2 on mssql
+
+Revision ID: 83f031fd9f1c
+Revises: a66efa278eea
+Create Date: 2020-07-23 12:22:02.197726
+
+"""
+
+from collections import defaultdict
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import mssql
+
+# revision identifiers, used by Alembic.
+revision = '83f031fd9f1c'
+down_revision = 'a66efa278eea'
+branch_labels = None
+depends_on = None
+
+
+def is_table_empty(conn, table_name):
+    """
+     This function checks if the mssql table is empty
+     :param conn: sql connection object
+     :param table_name: table name
+     :return: Booelan indicating if the table is present
+     """
+    return conn.execute('select TOP 1 * from {table_name}'.format(table_name=table_name)).first() is None
```

Comment:
I would really like to include this PR in 1.10.13 or 1.11.0 instead @aneesh-joseph if that is OK for you.

Mainly to reduce migrations or any chances of breaking and this PR seems to be touching multiple tables for MSSQL

Reply:
sure @kaxil , do you want me to base this PR against master ?

Reply:
Yes please